x/ccv/provider/types/params.go

```diff
@@ -21,11 +21,16 @@ const (
 
 	// DefaultMaxClockDrift defines how much new (untrusted) header's Time can drift into the future.
 	DefaultMaxClockDrift = 10 * time.Second
+
+	// DefaultTrustingPeriodFraction is the default fraction used to compute TrustingPeriod
+	// as UnbondingPeriod / TrustingPeriodFraction
+	DefaultTrustingPeriodFraction = 2
```

Comment:
We should change this to 3/2, cf. https://github.com/cosmos/ibc-go/blob/main/docs/architecture/adr-026-ibc-client-recovery-mechanisms.md, i.e., 
>  we recommend that the trusting period for the 07-tendermint client be set to 2/3 of the UnbondingPeriod.

Reply:
This variable is, and has been an integer. "fraction" is a bad previous naming for this variable, "denominator" would have been better. I've created [an issue](https://github.com/cosmos/interchain-security/issues/408) for this, but imo it'd be best to solve in a separate PR. As a parameter, it's not trivial to encode a concrete typed fraction into json, we could make two params for numerator and denominator maybe